ci/prow/deployments/deck.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         - --tide-url=http://tide/
         - --spyglass
         - --oauth-url=/github-login
+        - --cookie-secret=/etc/cookie/secret
```

Comment:
What is this cookie-secret? Is it also required to make this work?

Reply:
Sorry for not making it clear, this is a by product

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md
It was announced to be needed long time ago

Reply:
I see, nice!